### PR TITLE
added User Profile page

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Added my profile page of user where use can update their profile , can see their orders, can add or remove addresses.
FrontKart my profile page
FrontKart address page

### CODE REVIEW

**Comment by varun-Tad** [comment]
Looks good. You can go ahead and merge.

File: src/Components/Profile/Myprofile.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  padding: 5px;
+  outline: none;
+  margin: 8px;
+}
+/* input:invalid {
+  border-color: red;
+}
+input:valid {
+  border-color: var(--dark-success-color);
+} */
```
Comment (vaibhav18matere) [inline src/Components/Profile/Myprofile.css]: remove commented code if you are not going to use it later

File: src/Components/Profile/ProfilePage.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      link: "/myprofile/orders",
+    },
+    {
+      text: "Addresses",
+      link: "/myprofile/addresses",
+    },
+    // {
+    //   text: "Settings",
+    //   link: "/myprofile/settings",
+    // },
```
Comment (vaibhav18matere) [inline src/Components/Profile/ProfilePage.js]: remove commented code

File: src/Components/Profile/Setting.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+//   return (
+//     <>
+//       <div></div>
+//       <button className="btn primary-btn" onClick={logoutUser}>
+//         Logout
+//       </button>
+//     </>
+//   );
+// };
```
Comment (vaibhav18matere) [inline src/Components/Profile/Setting.js]: remove commented code

**Review by vaibhav18matere — APPROVED** [review]
- good file structure and readable code.
- naming convention is also top notch

**Pull request merged**